peachjam/models/judgment.py

```diff
@@ -583,15 +583,20 @@ class CauseList(CoreDocument):
         blank=True,
     )
     judges = models.ManyToManyField(Judge, blank=True, verbose_name=_("judges"))
+    end_date = models.DateField(_("end date"), null=True, blank=True)
 
     def assign_title(self):
-        court_name = self.registry.name if self.registry else ""
-        if not self.registry:
-            court_name = self.court.name if self.court else ""
-        nature_name = self.nature.name if self.nature else ""
-        date_str = self.date.strftime("%d %B %Y")
-
-        self.title = f"{court_name} - {nature_name} - {date_str}"
+        if not self.title:
```

Comment:
surely we want to update the title if any details change?

Reply:
I was thinking that an editor might want to change the title but I see that its incorrect